dts/bindings/sensor/adi,max6639-sensor.yaml

```
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2025 Tenstorrent AI ULC
+#
+# SPDX - License - Identifier : Apache - 2.0
+#
+
+description: |
+    ADI MAX6639 PWM Fan-Speed Controller, RPM sensor
+
+compatible: "adi,max6639-sensor"
```

Keep the Maxim branding, which is also consistent with what is upstream.
```suggestion
compatible: "maxim,max6639-sensor"
```